documentation/docs/test-suites/technical-interoperability/verifiable-credentials/index.md

```diff
@@ -7,10 +7,102 @@ import Disclaimer from './../../../\_disclaimer.mdx';
+
+   ```json
+   {
+     "@context": ["https://www.w3.org/ns/credentials/v2", "https://test.uncefact.org/vocabulary/untp/dpp/0.6.0/"],
+     "type": "EnvelopedVerifiableCredential",
+     "id": "data:application/vc+jwt,eyJhbGciOiJFZERTQSIsImtpZCI6..."
+   }
+   ```
+
+   Once confirmed, the credential is decoded for further processing.
+
+2. **Verify Signature**  
+   Confirm that the credential was issued by the claimed issuer, has not been tampered with, and is within the validity period if specified. This involves:
+
+   - Fetching the issuer's DID document using the `did:web` method, as specified in the [UNTP specification](https://uncefact.github.io/spec-untp/docs/specification/DigitalIdentityAnchor#did-methods).
+   - Obtaining the issuer's public key from the DID document.
+   - Verifying the credential's [Ed25519 signature](https://ed25519.cr.yp.to/) using the public key.
+   - Checking the `validFrom` and `validUntil` properties to ensure the credential is within its validity period.
+
+   If the credential has been tampered with, the public key is incorrect, or it is outside the validity period, the test will fail. These checks are abstracted within a [verifiable credential service](/docs/mock-apps/dependent-services/verifiable-credential-service.md) and exposed via an API endpoint (e.g., `http://localhost:3332/agent/routeVerificationCredential`).
+
+3. **Validate Schema Conformance**  
+   Ensure the decoded credential adheres to the [VCDM v2 schema](https://w3c.github.io/vc-data-model/schema/verifiable-credential/verifiable-credential-schema.json), verifying correct data types, required properties, and structural integrity.
+
+4. **Test JSON-LD Context Expansion**  
+   Expand the decoded credential's [JSON-LD context](https://www.w3.org/ns/credentials/v2) to confirm:
+
+   - All terms are defined.
+   - No terms conflict with protected terms.
+   - The expansion process completes without errors.
+
+5. **Check Credential Status**  
+   Verify the credential's status (e.g., active, revoked) using the value of `credentialStatus` within the credential, if present, ensuring the status method complies with the UNTP specification's permitted [BitstringStatusListEntry](https://www.w3.org/TR/vc-bitstring-status-list/) and that the credential has not been revoked. This process is abstracted behind an API endpoint of the [verifiable credential service](/docs/mock-apps/dependent-services/verifiable-credential-service.md) (e.g., `http://localhost:3332/agent/routeVerificationCredential`).
```

Comment:
Excellent, that whole section was actually really helpful for me (I'd not previously noted the explicit subset of functionality that UNTP is restricted to).